.github/CODEOWNERS

```diff
@@ -83,28 +98,40 @@ CMakeLists.txt @openvinotoolkit/openvino-ie-maintainers
 # OpenVINO Hetero plugin:
 /src/plugins/hetero/ @openvinotoolkit/openvino-hetero-maintainers
 
+# OpenVINO Template plugin:
+/src/plugins/template/ @openvinotoolkit/openvino-ie-template-maintainers
 
 # OpenVINO Frontends:
 /src/frontends/common/ @openvinotoolkit/openvino-common-frontend-maintainers
 /src/frontends/ir/ @openvinotoolkit/openvino-ir-frontend-maintainers
-/src/frontends/onnx/ @openvinotoolkit/openvino-onnx-frontend-maintainers
-/src/frontends/tests/onnx/ @openvinotoolkit/openvino-onnx-frontend-maintainers
 /src/frontends/paddle/  @openvinotoolkit/openvino-ie-paddle-maintainers
 /src/frontends/tensorflow/  @openvinotoolkit/openvino-tf-frontend-maintainers
 
-# OpenVINO Tests:
+# OpenVINO ONNX Frontend:
+/src/frontends/onnx/ @openvinotoolkit/openvino-onnx-frontend-maintainers
+/thirdparty/onnx/ @openvinotoolkit/openvino-onnx-frontend-maintainers @openvinotoolkit/openvino-ie-maintainers
+/thirdparty/protobuf/ @openvinotoolkit/openvino-onnx-frontend-maintainers @openvinotoolkit/openvino-ie-maintainers
```

Comment:
`protobuf` is not only for ONNX needed. TensorFlow needs it as well.

Reply:
please add tensorflow owners:)

Reply:
good catch, fixed